examples/grid_packing/main.rs

```diff
@@ -0,0 +1,41 @@
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+fn build_ui(application: &gtk::Application) {
+    // Create a new window, and set its title
+    let window = gtk::ApplicationWindow::new(application);
+    window.set_title(Some("Grid Packing"));
+
+    // Here we construct the grid that is going contain our buttons
+    let grid = gtk::Grid::new();
+
+    // Pack the grid in the window
+    window.set_child(Some(&grid));
+
+    // Create the first button and put it into the grid at (0, 0)
+    let button_1 = gtk::Button::with_label("Button 1");
+    button_1.connect_clicked(move |_| println!("Hello World"));
```

Comment:
```suggestion
    button_1.connect_clicked(move |_| eprintln!("Hello World"));
```

Reply:
"Hello World" is not an error, why print it to `stderr`?

Reply:
Oups, my bad